src/main/java/command/Command.java

```diff
@@ -0,0 +1,86 @@
+package command;
+
+import java.io.IOException;
+
+import exception.AnonymousException;
+import exception.DescriptionException;
+import exception.DukeDateTimeParserException;
+import exception.DukeKeywordException;
+import exception.NoIndexException;
+import storage.Storage;
+import task.TaskList;
+import ui.Ui;
+
+
+
+/**
+ * The Command class implements methods that will be used
+ * by its child class to create various Command type.
+ * It also contains strings of commands that specify
+ * the command type.
+ */
+public abstract class Command {
+    public static final String DONE_COMMAND = "done";
+    public static final String DELETE_COMMAND = "delete";
+    public static final String DEADLINE_COMMAND = "deadline";
+    public static final String EVENT_COMMAND = "event";
+    public static final String EXIT_COMMAND = "bye";
+    public static final String FIND_COMMAND = "find";
+    public static final String HELP_COMMAND = "--help";
+    public static final String LIST_COMMAND = "list";
+    public static final String TODO_COMMAND = "todo";
+    public static final String SHOW_AFTER_COMMAND = "show after";
+    public static final String SHOW_BEFORE_COMMAND = "show before";
+    public static final String DELETE_ALL_COMMAND = "delete all";
+    public static final String DONE_ALL_COMMAND = "done all";
+
+    private boolean isExit;
+
+    /**
+     * An empty constructor to initialize the Command object,
+     * and initialize isExit fields to false.
+     * isExit would be used to terminate the program when it is false.
```

Comment:
Comment seems to contain a little unnecessary/misplaced information. Information on the class members can be given using their own javadocs.